spec/integ/crypto/rust-crypto.spec.ts

```diff
@@ -237,6 +247,143 @@ describe("MatrixClient.initRustCrypto", () => {
         // The final call should have progress == total == -1
         expect(progressListener).toHaveBeenLastCalledWith(-1, -1);
     }, 60000);
+
+    describe("Legacy trust migration", () => {
+        beforeEach(() => {
+            fetchMock.reset();
```

Comment:
> I just added it these tests because they needed it and the other tests was fine so far without it

The problem with *not* doing a reset after each test is that it is easy to write tests that accidentally depend on mocks set up in a separate test, and then fixing it later is a nightmare.

But it's ok, no need to fix this now if you don't want to.

Reply:
changed to afterEach